src/IndentExtension.cpp

```diff
@@ -8,7 +8,7 @@
 static const int INDENT_SIZE = 4;
 
 static int findBulletSize(const QStringRef& ref) {
-    static QSet<QString> bullets = {"- ", "* ", "> "};
+    static QStringList bullets = {"- [ ] ", "- [x] ", "* [ ] ", "* [x] ", "- ", "* ", "> "};
```

Comment:
Is there a reason for this change?

Reply:
The loop will stop after the first match, and any line that matches `"- [ ] "` would also match `"- "`. Therefore, it is important to check for the task bullets first.